README.md

```diff
@@ -109,6 +111,10 @@ All the created API endpoints are documented and available [here](https://banka-
 * ```SALTING_ROUNDS```: Integer used to hash passwords
 * ```ADMIN_TOKEN, ADMIN_EMAIL, ADMIN_PASSWORD```: Admin token email and password for testing purposes
 
+### 3. Notes
+* When the server starts, a script creates the first admin user. He will be the one to create staff users and other admin users as well. 
```

Comment:
what credentials can a person using your app use to log in as that admin

Reply:
Oh!! I'm going to add the new ones. According to the previous comments.

Reply:
Thank you for the feedabacks